api/swagger-spec/oapi-v1.json

```diff
@@ -17874,6 +17874,10 @@
      "httpsProxy": {
       "type": "string",
       "description": "httpsProxy is a proxy used to reach the git repository over https"
+     },
```

Comment:
The problem we're trying to fix is that the default timeout is not applicable to all environments. In some cases we want a longer timeout. Also, the timeout may need to vary based on the git repo that you are cloning from.